etc/release/release.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -x
```

Comment:
set -ex belül a gitlabon hasznos volt, itt is hasonló a hatása, failel a job ha elszáll a docker push, nem tesszük be?